lib/db/mysql/scripts/purge-expired-tokens.sql

```diff
@@ -0,0 +1,43 @@
+DROP PROCEDURE IF EXISTS PURGE_EXPIRED_TOKENS;
+
+DELIMITER $$
+# Create a procedure that will delete a total of `roundCount` * `limitDeletePerRound` items, every `delaySeconds`.
+# Skip tokens that belong clientId
+CREATE PROCEDURE PURGE_EXPIRED_TOKENS(IN roundCount INT, IN limitDeletePerRound INT, IN delaySeconds INT, IN ignoreClientId CHAR(10))
```

Comment:
API usability nit: it may be better to let the caller pass in the desired maximum number of tokens to delete, rather than the number of rounds to execute.  This seems like the thing that the caller is most likely to want to control, so we shouldn't require them to calculate the values of `roundCount` and `limitDeletePerRound` that would achieve it.